src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,79 @@
 package core.basesyntax;
 
+import org.w3c.dom.Node;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static class Node<K, V> {
```

Comment:
inner class should be placed at the end of outer class
create constructor for all arguments